docs/docs/configuration.md

```diff
@@ -84,6 +84,37 @@ systemctl restart xo-server.service
 
 > For XO Proxy, the process is almost the same except the file to create is `/etc/systemd/system/xo-proxy.service.d/ca.conf` and the service to restart is `xo-proxy.service`.
 
+### Let's Encrypt support
```

Comment:
Let's Encrypt is obviously the most well known and used free public CA. 

But as we support others, I think we should mention this, maybe just in a `:::tip` box.